tests/models/test_dagbag.py

```diff
@@ -815,9 +815,9 @@ def test_sync_to_db_is_retried(self, mock_bulk_write_to_db, mock_s10n_write_dag,
         # Test that 3 attempts were made to run 'DAG.bulk_write_to_db' successfully
         mock_bulk_write_to_db.assert_has_calls(
             [
-                mock.call(mock.ANY, session=mock.ANY),
-                mock.call(mock.ANY, session=mock.ANY),
-                mock.call(mock.ANY, session=mock.ANY),
+                mock.call(mock.ANY, dag_directory=mock.ANY, session=mock.ANY),
+                mock.call(mock.ANY, dag_directory=mock.ANY, session=mock.ANY),
+                mock.call(mock.ANY, dag_directory=mock.ANY, session=mock.ANY),
```

Comment:
In this case I think this should be this?
```suggestion
                mock.call(mock.ANY, dag_directory=None, session=mock.ANY),
                mock.call(mock.ANY, dag_directory=None, session=mock.ANY),
                mock.call(mock.ANY, dag_directory=None, session=mock.ANY),
```